advanced/mysql-am/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "5.7"
 description: A Helm chart for WSO2 AM Datasources
 name: mysql-am
-version: 3.0.0-3
+version: 3.1.0-1
```

Comment:
@Chathuru please add the chart logo to MySQL based Helm chart as well.

Reply:
Fix in https://github.com/wso2/kubernetes-apim/pull/344/commits/33f03369f79ab2dbfe408b29cd8aaea47552b6c9 commit